frontend/src/services/PointsService.ts

```diff
@@ -36,6 +38,8 @@ export async function getAllInducteePoints(): Promise<InducteePoint[]> {
       hasSocialRequirement: point.hasSocialRequirement
         ? 'Complete'
         : 'Incomplete',
+      name: (await getUserById(point.user)).firstName + " " + (await getUserById(point.user)).lastName,
```

Comment:
each getUserByID is an API call - we should try to just call it once inside the map func

Reply:
fixed